### ISSUE TITLE
"pcdm" namespace is not registered by default

### ISSUE THREAD

**Issue by mjordan** [Islandora/documentation#1817]
The "pcdm" namespace needs to be registered by default. Here's a list of all registered namespaces in a vanilla Islandora:

![image](https://user-images.githubusercontent.com/403918/117460934-e367f080-af01-11eb-9ba9-11a8e5ae5f64.png)

**Comment by seth-shaw-unlv** [comment]
I spun up a fresh playbook using the ubuntu base and it looks fine:
<img width="780" alt="Screen Shot 2021-05-12 at 1 53 11 PM" src="https://user-images.githubusercontent.com/29869988/118043070-bf554680-b329-11eb-9806-3722c4d634b0.png">

Here is the JSON-LD config page:
<img width="584" alt="Screen Shot 2021-05-12 at 12 32 10 PM" src="https://user-images.githubusercontent.com/29869988/118042942-91700200-b329-11eb-81d0-6dcf25598301.png">

"rdau" and "pcdm" are both there.

How did you build your box, @mjordan?

**Comment by mjordan** [comment]
Weird - I built it using a standard `vagrant up` in the Playbook, on May 4. Given that date, could there have been updates to the relevant modules that would explain the missing namespaces in my build?

**Comment by seth-shaw-unlv** [comment]
The last playbook commit was on April 28th.

Perhaps you used the "islandora/8" base box (which is built on 1.1.0) instead of changing the base box to "ubuntu/focal64"? But then you shouldn't have had that report available, unless you checked out the most recent version of islandora_defaults after it completed provisioning. In that case, you need to update jsonld at the same time (well, jsonld's update needs to happen first).

**Comment by mjordan** [comment]
Right, I did build with "ubuntu/focal64", sorry for not indicating that earlier. However, I don't recall checking out any newer code after building. Let me build a new box on "ubuntu/focal64" and confirm no new code checkouts to see what happens.

**Comment by mjordan** [comment]
Ha, just cloned down a fresh copy of islandora-playbook and am seeing this error after `vagrant up`ping (and changing to 'ubuntu/focal64'):

```
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/home/mark/Documents/hacking/vagrants/islandora-playbook-newer/post-install.yml': line 13, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: Create a directory if it does not exist
      ^ here

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

`git diff` shows the following:

```
 git diff
diff --git a/Vagrantfile b/Vagrantfile
index 778a594..d656562 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -14,7 +14,8 @@ $virtualBoxDescription = ENV.fetch("ISLANDORA_VAGRANT_VIRTUALBOXDESCRIPTION", "I
 # Available boxes are 'islandora/8' and 'ubuntu/focal64'
 # Use 'ubuntu/focal64' to build a dev environment from scratch.
 # Use 'islandora/8' if you just want to download a ready to run VM.
-$vagrantBox = ENV.fetch("ISLANDORA_DISTRO", "islandora/8")
+# $vagrantBox = ENV.fetch("ISLANDORA_DISTRO", "islandora/8")
+$vagrantBox = ENV.fetch("ISLANDORA_DISTRO", "ubuntu/focal64")

 # vagrant is the main user
 $vagrantUser = "vagrant"
```

**Comment by seth-shaw-unlv** [comment]
The last update to post-install.yml was on April 8th. It worked fine for me just now. I have no idea why it is complaining to _you_. 😕🤷‍♂️

ansible version? I have 2.9.9.

**Comment by mjordan** [comment]
2.5.1. Let me upgrade it. Sigh.

**Comment by mjordan** [comment]
Which I can't do: "ansible is already the newest version (2.5.1+dfsg-1ubuntu0.1)".

This evening I'll build on my laptop that has what is probably a newer version of ansible.

**Comment by mjordan** [comment]
My build wth ansible 2.9.6 died at deploying Matomo again exactly as described in #1810. I'm too tired to try again, so based on your successful build, I'm happy to label this current issue as "can't replicate."
